Documentation/README_Button.md

```diff
@@ -149,6 +158,28 @@ To use your Icon Set select a button, expand the Icons dropdown in the `ButtonCo
 
 ![Button](../Documentation/Images/Button/MRTK_Button_Icon_Set_Assign.png)
 
+## How to change the size of a button
+
+HoloLens 2's shell-style button's size is 32x32mm. To customize the dimension, change the size of these objects in the button prefab:
+1. **FrontPlate**
+2. **Quad** under BackPlate
+3. **Box Collider** on the root
+
+Then, click **Fix Bounds** button in the NearInteractionTouchble script which is in the root of the button. 
+
+Update the size of the FrontPlate
+![Button](../Documentation/Images/Button/MRTK_Button_SizeCustomization1.png)
```

Comment:
Nit: I think that these paths can just be Images/Button/... right?

It's odd otherwise that it goes up a directory, then back into itself

Reply:
```suggestion
![Button](Images/Button/MRTK_Button_SizeCustomization1.png)
```